ci/jenkins/test.sh

```diff
@@ -375,6 +376,48 @@ function run_conformance {
     fi
 }
 
+function run_e2e_windows {
+    echo "====== Running Antrea e2e Tests ======"
+    export GO111MODULE=on
+    export GOPATH=${WORKDIR}/go
+    export GOROOT=/usr/local/go
+    export GOCACHE=${WORKDIR}/.cache/go-build
+    export PATH=$GOROOT/bin:$PATH
+    export KUBECONFIG=$KUBECONFIG_PATH
+
+    clean_for_windows_install_cni
+    kubectl apply -f build/yamls/antrea.yml
+    kubectl apply -f ${WORKDIR}/kube-proxy-windows.yml
+    kubectl apply -f build/yamls/antrea-windows.yml
+    kubectl rollout restart deployment/coredns -n kube-system
+    kubectl rollout status deployment/coredns -n kube-system
+    kubectl rollout status deployment.apps/antrea-controller -n kube-system
+    kubectl rollout status daemonset/antrea-agent -n kube-system
+    kubectl rollout status daemonset.apps/antrea-agent-windows -n kube-system
+    kubectl rollout status daemonset/kube-proxy-windows -n kube-system
+    kubectl get nodes -o wide --no-headers=true | awk -v role="$CONTROL_PLANE_NODE_ROLE" '$3 != role && $1 ~ /win/ {print $6}' | while read IP; do
+        for i in `seq 5`; do
+            sleep 5
+            timeout 5s ssh -o StrictHostKeyChecking=no -n Administrator@${IP} "powershell Get-NetAdapter -Name br-int -ErrorAction SilentlyContinue" && break
+        done
+        sleep 10
+    done
```

Comment:
it seems that this is duplicated code, should we put it in a separate function, e.g. `wait_for_antrea_windows_ready`?

Reply:
Good idea.